src/Settings.js

```diff
@@ -90,6 +92,23 @@
         }
     }
 
+    async getLocale() {
+        console.log('[Settings::getLocale]')
+        try {
+            const conf = await this._getRuntimeConfiguration()
+            const out = conf.userPreferences.locale
+            if (out == null) {
+                console.error('[Settings::getLocale] no locale in runtime config.userPreferences :(', conf)
+            }
```

Comment:
let's have a default here instead, it will make it a none-breaking change too. `en`